content/en/blog/2023/traffic-for-ambient-and-sidecar/index.md

```diff
@@ -0,0 +1,107 @@
+
+**(5) (6)**  After the request is sent to the  device pair (`veth httpbin <-> eth0 inside httpbin pod`),  the request would be intercepted and forwarded to the `istioin` device in the node B where httpbin pod is running by following its iptables rules and route rules. The `istioin` device in the node B is a `[geneve](https://www.rfc-editor.org/rfc/rfc8926.html)` tunnel, and the other side of this tunnel is pair to `pistioin` device which is inside the ztunnel pod of node B.
+
+**(7) (8)** After the request enters the `pistioin` device of the ztunnel pod, the request is intercepted to port 15008 of the ztunnel pod by the iptables rules of the ztunnel pod in node B.
+
+**(9)** The traffic getting into the port 15008 would be considered as a inbound request, then ztunnel will forward the request to the httpbin pod in the same node B.
+
+### [ sleep-> sidecar ] -> waypoint -> ztunnel -> helloworld httpbin network traffic path
+
+Comparing with the first part of the diagram, it's clear that the waypoint proxy path is added in path **"[ sleep -> sidecar ] -> ztunnel -> httpbin"**, and the extra configuration of waypoint proxy for helloworld service causes this.
+
+What's the background story?
+
+The Istiod control plane has all information of service and configuration in the service mesh. When helloworld is configured with waypoint proxy, the EDS configuration of helloworld service received by sidecar of sleep pod will be changed to type of `envoy_internal_address`, this causes that the request traffic going through the internal of envoy will be forwarded to port 15008 of waypoint proxy in node C via `[HBONE](https://docs.google.com/document/d/1Ofqtxqzk-c_wn0EgAXjaJXDHB9KhDuLe-W3YGG67Y8g/edit)` protocol.
+
+Similar with sidecar, waypoint proxy is based on envoy and it will forward the request to helloworld pod based on envoy L7 routing strategy.
+
+## Wrapping up
+
+Sidecar mode is significant to make Istio as a great service mesh solution, however, it also causes some problems because app and sidecar container must be in a pod together. Istio implements communication among services through centralized proxies (ztunnel and waypoint). This mode provides greater flexibility and scalability, reduces resource consumption, and allows more precise configuration based on demand. Therefore,  it's no doubt that Ambient mode is the evolution of Istio. However, Ambient is currently in alpha stage and is still on the way, and Sidecar mode is still the recommendation installation of Istio, so the coexistence of Sidecar and Ambient may be last for a while before "All in Ambient".
```

Comment:
```suggestion
Sidecar mode is significant to make Istio a great service mesh solution. However, it also causes some problems because app and sidecar containers must be in a pod together. Istio ambient mode implements communication among services through centralized proxies (ztunnel and waypoint). This mode provides greater flexibility and scalability, reduces resource consumption through not needing a sidecar for each pod in the mesh, and allows more precise configuration. Therefore,  it's no doubt that ambient mode is the next evolution of Istio. Currently, ambient is still alpha and sidecar mode is still the recommended mode of Istio. The coexistence of sidecar and ambient modes may be last for a while before we're "all in ambient".
```

Reply:
already done!